demos/src/demo-large-modifiers.mustache

```diff
@@ -4,7 +4,7 @@
 		<div class="o-teaser__content">
 			<div class="o-teaser__meta">
 				{{#article-tag-prefix}}<span class="o-teaser__tag-prefix">{{.}}</span>{{/article-tag-prefix}}
-				{{#article-genre}}<span class="o-teaser__genre">{{.}}</span>{{/article-genre}}
```

Comment:
where even did this come from?

Reply:
Perhaps there was an intention at some point to style tag types differently, who knows?!